### PR TITLE
Allowed network access in order to get assets from iCloud

### PR DESCRIPTION
**Pull request by gonzalobarrios**
Set property allowNetworkAsset in order to get assets from network (iCloud)
Set property synchronous to YES in order to wait for image load.

### CODE REVIEW

File: CTAssetsPickerController/CTAsset.m [hunk 1]
```diff
@@ -19,7 +19,8 @@ - (void)fetchImageWithTargetSize:(CGSize)targetSize
     PHImageRequestOptions *options = [[PHImageRequestOptions alloc] init];
 
     // make sure the call is asynchronous and only calls once the completion handler
-    options.synchronous = NO;
+    options.synchronous = YES;
```
Comment (gfzabarino) [inline CTAssetsPickerController/CTAsset.m]: Maybe since it’s synchronous now you could remove that part of the comment.

Comment (gonzalobarrios) [reply CTAssetsPickerController/CTAsset.m]: Do you mean to leave synchronous property with its default value? The default value is NO, so we need to specify the synchronous property to YES I think. What do you think?

Comment (gfzabarino) [reply CTAssetsPickerController/CTAsset.m]: No, I meant the comment above that, it says it’s asynchronous.

Comment (gonzalobarrios) [reply CTAssetsPickerController/CTAsset.m]: Ooh ok. Perfect, thanks.

**Pull request merged**